creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.ts

```diff
@@ -246,7 +253,8 @@ async function _handleIssueSyncRequest({
         ...syncRequestParameters,
         data: {
           ...syncRequestParameters.data,
-          forceResync: true,
+          forceResync: syncMode === SYNC_MODES.MergePrimaryAndSecondary,
+          forceWipe: syncMode === SYNC_MODES.MergePrimaryThenWipeSecondary,
           timestamp,
```

Comment:
ouch, kind of feels like i added more complexity by defining a separate boolean flag vs just passing in the `syncMode` here